src/hyperlight_host/src/mem/layout.rs

```diff
@@ -1100,7 +1099,7 @@ impl SandboxMemoryLayout {
 
         // Set up the security cookie seed
         let mut security_cookie_seed = [0u8; 8];
-        OsRng.fill_bytes(&mut security_cookie_seed);
+        OsRng.try_fill_bytes(&mut security_cookie_seed)?;
```

Comment:
I think maybe we should use `rand::rng().fill_bytes` here instead. In that way, we won't need rand_core